docs/en/04_Changelogs/4.0.0.md

```diff
@@ -212,6 +214,104 @@ instead, or if used in an actual XML file use `.CDATA` (see [template casting](/
 
 Where your code once used SQLQuery you should now use SQLSelect in all cases, as this has been removed (check the [3.2.0](3.2.0) upgrading notes).
 
+#### Upgrade code that uses i18n
+
+In many cases, localisation strings which worked in 3.x will continue to work in 4.0, however certain patterns
+have been deprecated and will be removed in 5.0. These include:
+
+ - _t calls with a $context parameter, which is ignored.
```

Comment:
OK - if there's a reason we're not using it that's no longer relevant, then maybe it's worth keeping.